app/views/contributions/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if project.stripe_connection.present? %>
+  <%= notification(t('.ready_to_take_payments')) %>
+<% else %>
+  <%= notification(t('.not_connected_to_payment_processor'), type: 'error') %>
```

Comment:
heads up, this doesn't match the styleguide, which uses `alert` instead of error. May need to add error if we want it. I made it alert to match the flash[:alert] in the layout; but happy to change it.